features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -316,6 +316,9 @@
             <ScopeValidator class="org.wso2.carbon.identity.oauth2.validators.xacml.XACMLScopeValidator"/>
         </ScopeValidators>
 
+        <!-- Uncomment below to enable role based scope validation for implicit grant. -->
+        <!--<RoleBasedScopeValidationEnabled>true</RoleBasedScopeValidationEnabled>-->
```

Comment:
Why can't we by default enable the role-based scop validation for upcoming releases and have a property to disable the feature if anyone wants the previous behavior?

Reply:
+1

Reply:
done